### DISCUSSION TITLE
"Closed" test for Google Play publishing

### DISCUSSION THREAD

**Discussion by amberin** [orgzly-revived/orgzly-android-revived#250] [Announcements]
Related issue: #35 

I have published the app in the Google Play Console and initiated a "closed test". We need 20 people testing for 14 days before we can move on to the next step in the process.

Let's use this thread to discuss this testing process and solve any issues that arise.

To participate, first join the Google Group orgzly-revived-testers@googlegroups.com. The group is open for anyone to join, you should be able to search for it in the Google Groups interface. Members of this group will have access to the closed test.

Join the testing via Google Play: https://play.google.com/store/apps/details?id=com.orgzlyrevived
Join the testing via the web browser: https://play.google.com/apps/testing/com.orgzlyrevived

<strike>I need to finish some paper work in the Play Console before the testing can start, but I will update here once it's finished.</strike>

Update: The latest beta build has now been approved and you should be able to join the test by clicking one of the links above. Note that membership in the Google Group "orgzly-revived-testers" is required, but anyone with a Google account can join the group.

**Comment by amberin** [comment]
You are supposed to be able to join the test and install the app by following this link: https://play.google.com/apps/internaltest/4701342774943410653. Please try it out -- we need as many people as possible.

**Reply by dwoffinden** [reply]
I see

> App not available for this account
> Your account hasn't yet been invited to participate in this app's internal testing program.

**Reply by amberin** [reply]
*scratches head*

In Play Console, the only really obvious way to invite individuals is by building an email list. But surely, there must be some other way? I have no interest in collecting everyone's personal information.

But I suppose anyone who wants to is welcome to shoot an email to orgzlyrevived@gmail.com, and I will let you know when I have added your address to the list. **Please note that the address you provide must be linked to a Google account.**

**Reply by amberin** [reply]
Ah, I had chosen "internal test" instead of "closed test". Now that I am initiating a "closed test", I can specify a Google Group. I will create one.

**Comment by cashpw** [comment]
I see "A testing version of this app hasn't been published yet or isn't available for this account." when I follow the web link. I'm a member of the [Google group](https://groups.google.com/g/orgzly-revived-testers). Is the testing build published? Maybe there's a permissions issue?

**Comment by amberin** [comment]
Yeah, sorry. I'm waiting for review of all the info before the testing can start, but I will update here once it's finished.

**Comment by amberin** [comment]
We were just rejected due to this: https://github.com/orgzly-revived/orgzly-android-revived/issues/21#issuecomment-2111608244

We'll need to do something about that and try again.

**Comment by amberin** [comment]
Good news! The latest beta build has now been approved and you should be able to join the test by clicking one of the links in the description.

**Reply by cashpw** [reply]
Working for me! Thank you!

**Comment by amberin** [comment]
I just released v1.8.23-beta.1 to Play store testers. Please let me know if it is not available to you within a few days.

**Comment by erikmd** [comment]
Hi! thanks for your efforts! — Question:
> We need 20 people testing for 14 days before we can move on to the next step in the process.

are you enough people for the test? what are the latest news w.r.t. publishing the app on Google Play?

**Reply by amberin** [reply]
Yes, we have now surpassed 20 testers, and we will apply for production status today.

**Reply by erikmd** [reply]
Awesome! thanks for this great news 👍

**Comment by amberin** [comment]
We are now live in the Play store. 🥳
